hypha/public/forms/models.py

```diff
@@ -42,7 +42,7 @@ class FormField(AbstractFormField):
 
 class ExtendedFormBuilder(FormBuilder):
     def create_document_field(self, field, options):
-        return FileField(**options)
+        return FileField(widget=FileInput(attrs={'accept': settings.FILE_ACCEPT_ATTR_VALUE}), **options)
```

Comment:
I changed to FILE_ACCEPT_ATTR_VALUE so the formatting gets correct for the accept attribute.